pallet/account-migration/src/lib.rs

```diff
@@ -497,20 +496,6 @@ pub mod pallet {
 					)?;
 				}
 
-				let k = l.staked_kton + l.unstaking_kton.iter().map(|(k, _)| k).sum::<Balance>();
-
-				// To calculated the worst case in benchmark.
-				debug_assert!(k > 0);
-
-				if k != 0 {
-					<pallet_assets::Pallet<T>>::transfer(
-						RawOrigin::Signed(*to).into(),
-						KTON_ID.into(),
-						staking_pot,
-						k,
-					)?;
-				}
-
 				<darwinia_staking::Ledgers<T>>::insert(to, l);
```

Comment:
!!Be careful here:
The Ledger value type in staking is different with the Old Ledger in the account-migration. The Ledger for 1.0 storage is still using Old Ledger. So it can not use the ledger type from Staking anymore, other wise data storage reading will have issues.